import-export-cli/docs/apictl_set.md

```diff
@@ -28,7 +28,7 @@ apictl set --mode default
 ### Options
 
 ```
-      --export-directory string    Path to directory where APIs should be saved (default "/home/wasura/.wso2apictl/exported")
+      --export-directory string    Path to directory where APIs should be saved (default "/home/naduni/.wso2apictl/exported")
```

Comment:
is it possible to use a generic name such as "wso2" here instead of the specific names?

Reply:
This is an auto generated page. So I don't think it's possible to change this. Do you have any idea on how to do that?

Reply:
Ah okay. I was just wondering :)